.travis.yml

```diff
@@ -6,6 +6,8 @@ sudo: false
 cache:
 - node_modules
 - yarn
+before_install:
+  - sudo apt -y install p7zip-full
```

Comment:
What is the purpose of p7zip here?

Reply:
note that we have sudo: false on our travis.yml also

Reply:
Looks like `sudo: false` is deprecated (and probably not doing anything since we use Xenial): https://docs.travis-ci.com/user/reference/overview/#deprecated-virtualization-environments

p7zip is needed to build the CLI package

Reply:
interesting. maybe we'll remove the sudo false then